thread_setup_windows.c

```diff
@@ -0,0 +1,31 @@
+#include "goopenssl.h"
```

Comment:
Condition for consistency's sake?

```suggestion
//go:build windows

#include "goopenssl.h"
```